frontend/src/components/Header.stories.js

```diff
@@ -1,11 +1,12 @@
 import Header from './Header';
-import {fn} from '@storybook/test'
+import { action } from '@storybook/addon-actions';
 
```

Comment:
what are you trying to do here ?

I didn't even know about this addon for storybook but it turns out it's because this is [already installed by default in the current version](https://storybook.js.org/addons/@storybook/addon-actions)

Don't make changes you don't understand or have a reason for - this is doing the same thing `fn` already did but your change introduces a new dependency that is deprecated.